contracts/senders/aave-v2-migrator/interfaces.sol

```diff
@@ -1,6 +1,20 @@
 pragma solidity >=0.7.0;
 pragma experimental ABIEncoderV2;
 
+struct ReserveConfigurationMap {
+    //bit 0-15: LTV
+    //bit 16-31: Liq. threshold
+    //bit 32-47: Liq. bonus
+    //bit 48-55: Decimals
+    //bit 56: Reserve is active
+    //bit 57: reserve is frozen
+    //bit 58: borrowing is enabled
+    //bit 59: stable rate borrowing enabled
+    //bit 60-63: reserved
+    //bit 64-79: reserve factor
+    uint256 data;
+}
+
```

Comment:
```suggestion
```

Reply:
What is this?

Reply:
Bit map of the different parameters stored for an asset. We can remove this as we are not using this anywhere